### PR TITLE
Feat(loading): Loading background while waiting for decentralised renderer

### PR DESCRIPTION
**Pull request by samuellye98**
https://github.com/OpenCerts/opencerts-website/issues/359

### CODE REVIEW

File: src/components/CertificateViewer.js [hunk 1]
```diff
@@ -28,6 +28,15 @@ const renderVerifyBlock = props => (
   />
 );
 
+const LoadingIframe = (
```
Comment (yehjxraymond) [inline src/components/CertificateViewer.js]:
```suggestion
const LoadingIframe = () =>
```

File: src/components/CertificateViewer.js [hunk 2]
```diff
@@ -91,14 +100,17 @@ const CertificateViewer = props => {
...
-      <DecentralisedRenderer
-        certificate={document}
-        source={`${
-          typeof document.data.$template === "object"
-            ? certificate.$template.url
-            : LEGACY_OPENCERTS_RENDERER
-        }`}
-      />
+      <div>
+        {LoadingIframe}
```
Comment (yehjxraymond) [inline src/components/CertificateViewer.js]:
```suggestion
        <LoadingIframe/>
```

File: src/components/CertificateViewer.js (hunk 1 above)
Comment (Nebulis) [reply src/components/CertificateViewer.js]: I would suggest to use prettier :D

File: src/components/CertificateViewer.js [hunk 3]
```diff
@@ -91,14 +100,17 @@ const CertificateViewer = props => {
...
-        certificate={document}
-        source={`${
-          typeof document.data.$template === "object"
-            ? certificate.$template.url
-            : LEGACY_OPENCERTS_RENDERER
-        }`}
-      />
+      <div>
+        <LoadingIframe />
+        <DecentralisedRenderer
```
Comment (Nebulis) [inline src/components/CertificateViewer.js]: What do you think if the renderer send a signal saying it's loaded ?

(to do in another MR)

Comment (samuellye98) [reply src/components/CertificateViewer.js]: like a notification?

Comment (Nebulis) [reply src/components/CertificateViewer.js]: yeah for instance, like the setHeight but just for the status, maybe instead of setHeight we should switch to an action dispatcher a la redux =)

Comment (Nebulis) [reply src/components/CertificateViewer.js]: Current code is (in document-renderer)

```
window.connection = window.Penpal.connectToChild({
      iframe,
      methods: {
        updateHeight,
        updateTemplates 
      }
    });
```

Suggestion is (sth like)

```
const actions = function(action) {
      if (action.type === 'SET_HEIGHT') {
        updateHeight(action.payload);
      } else if (action.type === 'SET_TEMPLATES') {
        updateTemplates(action.payload);
      } else {
        console.log(`I dont handle ${action.type}`)
      }
    };
    window.connection = window.Penpal.connectToChild({
      iframe,
      methods: {
        actions
      }
    });
```

Again out f scope of this MR =)

**Review by Nebulis — APPROVED** [review]
